### PR TITLE
Newsletter 224: Add german translation

### PR DESCRIPTION
**Pull request by tschweg**
Also contains minor fixes to the German style guide.

### CODE REVIEW

File: _posts/de/newsletters/2022-11-02-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+    Vor einigen Jahren schlug Joost Jager eine Lösung vor (siehe
+    [Newsletter #51][news51 attrib]), die er nun mit Verbesserungen und
+    zusätzlichen Details [aktualisiert][jager attrib] hat. Der Mechanismus würde
+    die Identifikation des Knotenpaars sicherstellen, zwischen dem
+    eine Zahlung fehlgeschlagen ist (oder zwischen dem eine frühere
+    Fehlermeldung zensiert oder unverständlich wurde). Der größte Nachteil von
+    Jagers Vorschlag besteht darin, dass er die LN-Onion-Nachrichtengrösse
+    für Fehlschläge deutlich erhöhen würde, wenn andere LN-Eigenschaften gleich
+    blieben; obwohl bei Fehlschlägen die Onion-Nachrichtengrösse nicht so
+    groß sein müsste, wenn die maximale Anzahl der LN-Hops verringert würde.
```
Comment (tschweg) [inline _posts/de/newsletters/2022-11-02-newsletter.md:128]: The phrasing here is rather bumpy, but I couldn't come up with anything better.

File: _posts/de/newsletters/2022-11-02-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+[Core Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], und [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #23927][] schränkt `getblockfrompeer` auf beschnittenen (pruned)
+  Knoten auf Blockhöhen unterhalb des aktuellen Synchronisationsfortschritts des
+  Knotens ein. Dies verhindert, dass ein Footgun entsteht, der zukünftige Blöcke
+  abruft und damit die Blockdateien des Knotens für das Pruning ungeeignet macht.
```
Comment (tschweg) [inline _posts/de/newsletters/2022-11-02-newsletter.md:193]: _beschnitten_ is suboptimal for pruning, but I couldn't fine a better German term. As a noun, I would suggest to keep _Pruning_.

File: _posts/de/newsletters/2022-11-02-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+  Wallet relevanten UTXOs ausgeben oder erzeugen.
+
+- [Bitcoin Core #23578][] nutzt [HWI][topic hwi] und die kürzlich eingefügte
+  Unterstützung für [BIP371][] (siehe [Newsletter #207][news207 bc22558]), um
+  externe Signierunterstützung für [Taproot][topic taproot] Schlüsselpfadausgaben
+  zu ermöglichen.
+
+- [Core Lightning #5646][] aktualisiert die experimentelle Implementierung von
+  [Angeboten][topic offers], um [x-only öffentliche Schlüssel][news72 xonly] zu
+  entfernen (stattdessen werden [komprimierte Pubkeys][compressed pubkeys]
```
Comment (tschweg) [inline _posts/de/newsletters/2022-11-02-newsletter.md:215]: Unclear what the colloquial abbreviation for _öffentliche Schlüssel_ is. Pubkeys should work though.

**Comment by tschweg** [comment]
@Xekyo : Would you be so kind to do the usual review? Thank you in advance. :bow:

**Comment by tschweg** [comment]
@Xekyo, @bitschmidty : Könntet ihr auf die Ausgabe einen Blick werfen? Vielen Dank!

File: _posts/de/newsletters/2022-11-02-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+[wuille bip324]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/021115.html
+[news72 xonly]: /en/newsletters/2019/11/13/#x-only-pubkeys
+[compressed pubkeys]: https://developer.bitcoin.org/devguide/wallets.html#public-key-formats
+[blinded payments]: /en/topics/rendez-vous-routing/
+[news104 intercept]: /en/newsletters/2020/07/01/#lnd-4018
+[news51 attrib]: /en/newsletters/2019/06/19/#authenticating-messages-about-ln-delays
+[jager attrib]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2022-October/003723.html
+[russell attrib]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2022-October/003727.html
+[teinturier fees]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2022-October/003729.html
+[news222 rbf]: /en/newsletters/2022/10/19/#transaction-replacement-option
```
Comment (bitschmidty) [inline _posts/de/newsletters/2022-11-02-newsletter.md:273]: I believe this line can be removed as its duplicated above

File: _posts/de/newsletters/2022-11-02-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+[news223 rbf]: /en/newsletters/2022/10/26/#continued-discussion-about-full-rbf
```
Comment (bitschmidty) [inline _posts/de/newsletters/2022-11-02-newsletter.md:274]: I believe this line can be removed as /de/ version listed above

File: _posts/de/newsletters/2022-11-02-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+{% include linkers/issues.md v=2 issues="23927,25957,5646,6517,7001,6831,957,779,1036,845,1378,23578,22558" %}
+[bitcoin core 24.0 rc2]: https://bitcoincore.org/bin/bitcoin-core-24.0/
+[bcc testing]: https://github.com/bitcoin-core/bitcoin-devwiki/wiki/24.0-Release-Candidate-Testing-Guide
+[lnd 609cc8b]: https://github.com/LightningNetwork/lnd/commit/609cc8b883c7e6186e447e8d7e6349688d78d4fd
+[lnd secpol]: https://github.com/lightningnetwork/lnd/security/policy
+[towns consistency]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/021116.html
+[news205 rbf]: /en/newsletters/2022/06/22/#full-replace-by-fee
+[news208 rbf]: /en/newsletters/2022/07/13/#bitcoin-core-25353
+[news222 rbf]: /en/newsletters/2022/10/19/#transaction-replacement-option
+[news223 rbf]: /de/newsletters/2022/10/26/#continued-discussion-about-full-rbf
```
Comment (bitschmidty) [inline _posts/de/newsletters/2022-11-02-newsletter.md:263]:
```suggestion
[news223 rbf]: /de/newsletters/2022/10/26/#fortsetzung-der-diskussion-uber-vollstandige-rbf
```

**Review by bitschmidty — COMMENTED** [review]
formatting and links checked. couple suggestions.

File: _posts/de/newsletters/2022-11-02-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,277 @@
...
+## Nennenswerte Code- und Dokumentationsänderungen
+
+*Erwähnenswerte Änderungen diese Woche in [Bitcoin Core][bitcoin core repo],
+[Core Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], und [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #23927][] schränkt `getblockfrompeer` auf beschnittenen (pruned)
```
Comment (Xekyo) [inline _posts/de/newsletters/2022-11-02-newsletter.md:190]: Beschnitten weckt bei mir andere Assoziationen. Vielleicht "zurückgeschnittenen" oder "zurechtgestutzten"?

```suggestion
- [Bitcoin Core #23927][] schränkt `getblockfrompeer` auf zurechtgestutzten (pruned)
```

File: _posts/de/newsletters/2022-11-02-newsletter.md (hunk 2 above)
Comment (Xekyo) [reply _posts/de/newsletters/2022-11-02-newsletter.md:193]: Auf den zweiten Blick, vielleicht bleiben wir einfach dabei: 
image

Comment (Xekyo) [inline _posts/de/newsletters/2022-11-02-newsletter.md:193]:
```suggestion
  Knotens ein. Dies verhindert, dass ein Footgun entsteht, bei dem das Abrufen unsynchronisierter Blöcke
  die Blockdateien des Knotens für Pruning ungeeignet macht.
```

File: _posts/de/newsletters/2022-11-02-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,277 @@
...
+  verwendet, die ein zusätzliches Byte enthalten). Er implementiert auch die
+  Weiterleitung von [verdeckten Zahlungen][blinded payments]; ein weiteres
+  experimentelles Protokoll. In der PR-Beschreibung wird davor gewarnt, dass er
+  "nicht die Erstellung und tatsächliche Bezahlung von Rechnungen mit verdeckten
+  Zahlungen beinhaltet."
+
+- [LND #6517][] fügt eine neue RPC und einen Event hinzu, die es einem Benutzer
+  ermöglichen, zu überwachen, wann eine eingehende Zahlung ([HTLC][topic htlc])
+  vollständig gesperrt ist, indem die Verpflichtungstransaktion aktualisiert
+  wird, um die neue Channel-Saldoverteilung widerzuspiegeln.
```
Comment (Xekyo) [inline _posts/de/newsletters/2022-11-02-newsletter.md:225]:
```suggestion
  abgeschlossen ist, nachdem die Verpflichtungstransaktion aktualisiert
  wurde, um die neue Channel-Saldoverteilung widerzuspiegeln.
```

**Review by Xekyo — COMMENTED** [review]
Sorry for the slow review, I was on vacation

**Comment by bitschmidty** [comment]
@tschweg can you take a look at @Xekyo 's comments?

**Comment by tschweg** [comment]
@bitschmidty : Absolutely. Sorry for the delay, I was rather swamped the last 2 weeks. I'll have the corrections in within the next 1-2 days.

**Comment by Xekyo** [comment]
Yeah, I feel you. It also took me forever to finish reviewing the PR. It would be nice if we had a couple more people look at it with us.

**Review by Xekyo — APPROVED** [review]
LGTM, tested all the links and looked it over in the preview build.

**Comment by tschweg** [comment]
So much for 1-2 days :sweat_smile: But we should be good now with the changes. Could I trouble you for a final review agian, @Xekyo

**Comment by Xekyo** [comment]
I only read the changes again, but they looked good to me. About to merge :)

**Comment by Xekyo** [comment]
Rebased on master

**Comment by Xekyo** [comment]
Thanks for the long wind!

**Pull request merged**